interface/src/app/gene/gene-search-form.tpl.html

```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="col-xs-12">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <div class="col-xs-3">
+          <h4>Search for Genes</h4>
+          <form class="form">
+            <textarea name="genes" rows="7" ng-model="genesToAdd.query"
+              placeholder="Search genes by symbol or Entrez ID">
+            </textarea>
+            <span class="error">{{ errors.name }}</span>
+            <br>
+            <button class="btn btn-primary" ng-click="searchGenes()"
+              ng-disabled="loadingSearchResults">
+              Search   <i class="fa fa-search"></i>
+            </button>
+          </form>
+        </div>
+        <div class="col-xs-9">  
+          <search-result-table></search-result-table>
```

Comment:
Ok, I think this works for now, but we can play around a little more with the layout and see if we want to change it afterwards.